db/migrate/20220823145959_create_filter_tag.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateFilterTag < ActiveRecord::Migration[6.1]
+  def change
+    create_table :filter_tags do |t|
```

Comment:
Not sure about the table name here. I don't have a lot of great ideas for what to rename it to, though. Maybe `filter_tag_categories` or `filter_tag_category_mappings`?